### DISCUSSION TITLE
[A-L] (2023/24) Foglio 11 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#219] [Esercizi - Bertini]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/6cac0379-fbe3-4e07-90f1-8c3a3f983219)

**Comment by IonutCicio** [comment]
La catena di Markov che rappresenta il problema è la seguente

<img width="400" alt="Markov chain" src="https://github.com/sapienzastudentsnetwork/calcprob2324/assets/40034429/235b8f4d-443d-4d3b-baef-b7c96d749783">

Quindi si ha la probabilità di transizione e probabilità dello stato iniziale

$$
P = 
\begin{pmatrix}
\frac{1}{2} & \frac{1}{2}\\
\frac{3}{10} & \frac{7}{10}
\end{pmatrix}
\\\\,
\mu = \begin{pmatrix} 0 & 1 \end{pmatrix}
$$

#### 1)

Per calcolare la probabilità che mercoledì piova bisogna trovare la probabilità di transizione dopo 2 passi, quindi $P^2$, e moltiplicarlo per il vettore dello stato iniziale

$$
\mu P^2 =  
\begin{pmatrix} 0 & 1 \end{pmatrix}
\begin{pmatrix}
\frac{2}{5} & \frac{3}{5}\\
\frac{9}{25} & \frac{16}{25}
\end{pmatrix}
= \begin{pmatrix} \frac{9}{25} & \frac{16}{25} \end{pmatrix}
$$

Quindi la probabilità che mercoledì piova è $\frac{9}{25}$

#### 2)

Per calcolare la probabilità che giovedì **non** piova bisogna trovare la probabilità di transizione dopo 3 passi, quindi $P^3$, e moltiplicarlo per il vettore dello stato iniziale

$$
\mu P^3 =  
\begin{pmatrix} 0 & 1 \end{pmatrix}
\begin{pmatrix}
\frac{19}{50} & \frac{31}{50}\\
\frac{93}{250} & \frac{157}{250}
\end{pmatrix}
= \begin{pmatrix} \frac{93}{250} & \frac{157}{250} \end{pmatrix}
$$

Quindi la probabilità che mercoledì piova è $\frac{157}{250}$

#### 3)

Per calcolare la probabilità che sia mercoledì sia giovedì piova, basta calcolare la probabilità che mercoledì piova (ovvero $\frac{9}{25}$ e moltiplicarla per $\frac{1}{2}$ (la probabilità che il giorno successivo ripiova, quindi $\frac{9}{50}$
